docs/career_assessment_how_to.md

```diff
@@ -0,0 +1,65 @@
+# Career assessment
+## How does it work?
```

Comment:
I would give the one-liner about what Career Assessment is to set the context for the follow-up question "How does it work?"

example: The career assessment is a questionnaire that helps to determine how your traits match with current job opportunities.